.github/workflows/contract-tests.yml

```diff
@@ -52,6 +60,20 @@ jobs:
         with:
           version: nightly
 
+      - name: Prepare Environment Variables
+        run: |
+          cp .env.sample.goerli .env
+          export $(grep -v '^#' .env | xargs) # Load variables
+          for var in $(grep -v '^#' .env | cut -d= -f1); do
+            echo "$var=${!var}" >> $GITHUB_ENV
```

Comment:
Not very important but isn't this the same as `grep -v '^#' .env >> $GITHUB_ENV`?

Reply:
https://github.com/EspressoSystems/nitro-contracts/pull/24/commits/c1782e80b4ab0f0b3746b06a479fe53450c131dd